client/js/components/AccountLoginForm/AccountLoginForm.js

```diff
@@ -47,16 +55,14 @@ class AccountLoginForm extends Component {
             <Form
               onSubmit={async values => {
                 try {
-                  const data = await login({
+                  const newUserToken = await login({
                     variables: {
                       email: values.email,
                       password: values.password,
                     },
                   });
-
-                  if (data) {
-                    this.props.navigation.navigate('Home');
-                  }
+                  await createViewer(newUserToken.data.login);
+                  this.props.navigation.navigate('Home');
                 } catch (e) {
                   console.log(e);
```

Comment:
Display errors if any for your user

Reply:
This is done. Error text added to sign up and login form based on errors.